src/oatpp/web/server/AsyncHttpConnectionHandler.hpp

```diff
@@ -37,8 +37,8 @@ namespace oatpp { namespace web { namespace server {
 class AsyncHttpConnectionHandler : public base::Countable, public network::ConnectionHandler {
 private:
   std::shared_ptr<oatpp::async::Executor> m_executor;
-private:
   std::shared_ptr<HttpProcessor::Components> m_components;
+  std::shared_ptr<std::atomic_ulong> m_spawns;
```

Comment:
Maybe it's better to store a plain atomic without shared_ptr wrapper - to avoid double atomic incs and decs
one for ptr counter and one for spawns

Reply:
I was thinking the same but couldn't decide between clean pointer passing or performance. But performance is key at this point, you are right.